torch/_inductor/graph.py

```diff
@@ -194,6 +194,8 @@ def __init__(
         self.unaligned_buffers: Set[str] = set()
         self.name_to_buffer: Dict[str, ir.ComputedBuffer] = {}
         self.name_to_users: DefaultDict[str, List[ir.IRNode]] = defaultdict(list)
+        self.name_to_coerced_dtype: Dict[str, torch.dtype] = {}
+        self.data_dep_symbol_to_expr_str: Dict[str, str] = {}
```

Comment:
Some docs for these would be appreciated

Reply:
removed the need for `data_dep_symbol_to_expr_str`